spec/controllers/your_results_controller_spec.rb

```diff
@@ -28,6 +29,19 @@
 
   describe '#schedule' do
     it 'returns an array of formatted PeriodContributions' do
+      request.session[:salary] = 75
+      request.session[:salary_frequency] = 'week'
+
+      expect(Wpcc::SalaryPerYear)
+        .to receive(:new)
+        .with(
+          salary: 75,
+          salary_frequency: 'week'
+        )
+        .and_return(salary_per_year)
```

Comment:
@margOnline in this spec we are checking for a type. I'd rather that we checked that the conversion actually returns a real value rather than the type of object returned. Wdyt?

Reply:
I'm not sure I agree. There are already tests to ensure that the SalaryPerYear class and particularly the convert method returns the correct value.

Reply:
if `Wpcc::SalaryPerYear` is already tested well then I agree that this current test is enough

Reply:
are there any 'sad' path tests that need to be written?

Reply:
If it's already well-tested that the conversions works well then I am happy with it